docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+function do_help {
+  echo HELP:
+  echo "Supported commands:"
+  echo "   master              - Start a Kudu Master"
+  echo "   tserver             - Start a Kudu TServer"
+  echo "   help                - print useful information and exit"
+  echo ""
+  echo "Other commands can be specified to run shell commands."
+
+  exit 0
+}
+
+if [ "$1" = 'master' ]; then
+	exec kudu-master -fs_wal_dir /data/kudu-master --logtostderr \
+  --use_hybrid_clock=false
+elif [ "$1" = 'tserver' ]; then
+  exec kudu-tserver -fs_wal_dir /data/kudu-tserver \
+  -tserver_master_addrs ${KUDU_MASTER} \
+  --logtostderr --use_hybrid_clock=false
+elif [ "$1" = 'help' ]; then
+  do_help
+fi
+
```

Comment:
The idea was that the Dockerfile's default command is 'help' to show possible options. master or tserver can be specified to launch with the default arguments, but you can still pass any custom command (as in the original, since in that case exec "$@" is invoked).